README.md

```diff
@@ -1,9 +1,27 @@
+
 # Minionify
-An app that allows basic control over remote devices.
 
+## Background
+An app that allows basic control over remote devices.
 The users would be divided into 2 groups; managers and minions.
 A manager can use the app to execute certain operations on the minions' devices, such as gathering statistics, sending notifications, determining location, opening another app etc.
-
 The minions would use the app passively; by using the app, they willingly give the managers permission to execute operations on their devices.
+Each minion can have only one manager, while managers can multiple minions.
 
 Possible users of the app are worried parents with their children or strict bosses with their employees.
+
+
+## Roles
+- Minions: The minions will run a background service that will listen for incoming requests from the managers.
+
+- Managers: The managers will have a GUI that lets them control their minions.
```

Comment:
Confusing contextual-use of Minions and Managers. The minions you mention 2 lines above are from the app's perspective - they listen to requests. The managers on the other hand are from the user's perspective (GUI). Be consistent